manifests/params.pp

```diff
@@ -25,6 +25,7 @@
     Integer $notifications_sets          = lookup('gb_backend::notifications_sets', Integer, first, 20),
     Integer[0,23] $notifications_trigger_hour = lookup('gb_backend::notifications_trigger_hour', Integer[0,23], first, 0),
     Integer[0,59] $notifications_trigger_minute = lookup('gb_backend::notifications_trigger_minute', Integer[0,59], first, 0),
+    Optional[String] $notifications_client_app_name = lookup('gb_backend::notifications_client_app_name', Optional[String], first, 'Glowing Bear'),
```

Comment:
I guess it can be type String instead of Optional[String], because there is a default value. The value will never be `undef`.

Reply:
thanks, I'll change it